databricks_blueprints/errors.py

```diff
@@ -3,6 +3,14 @@
 
 EXIT_CODE_INVALID_CREDENTIALS = 200
 EXIT_CODE_INVALID_INSTANCE = 201
+EXIT_CODE_BAD_REQUEST = 202
```

Comment:
This is a copy of EXIT_CODE_INVALID_CLUSTER. 
You also have multiple names for the 0 value. You can create just one constant and use it in multiple places indicating success.